test/unit/UFragmentsPolicy.js

```diff
@@ -101,62 +114,57 @@ contract('UFragments:setMarketOracle:accessControl', function (accounts) {
   });
 });
 
-contract('UFragmentsPolicy:setDeviationThreshold', async function (accounts) {
-  let prevThreshold, threshold;
-  before('setup UFragmentsPolicy contract', async function () {
-    await setupContracts();
-    prevThreshold = await uFragmentsPolicy.deviationThreshold.call();
-    threshold = prevThreshold.plus(0.1 * 10 ** 18);
-    await uFragmentsPolicy.setDeviationThreshold(threshold);
-  });
+contract('UFragmentsPolicy:setCpiOracle', async function (accounts) {
+  before('setup UFragmentsPolicy contract', setupContracts);
 
-  it('should set deviationThreshold', async function () {
-    (await uFragmentsPolicy.deviationThreshold.call()).should.be.bignumber.eq(threshold);
+  it('should set cpiOracle', async function () {
+    await uFragmentsPolicy.setCpiOracle(deployer);
+    expect(await uFragmentsPolicy.cpiOracle.call()).to.eq(deployer);
   });
 });
 
-contract('UFragments:setDeviationThreshold:accessControl', function (accounts) {
+contract('UFragments:setCpiOracle:accessControl', function (accounts) {
```

Comment:
Should this be 'UFragmentsPolicy:setCpiOracle:accessControl'?

Reply:
I've added one test case and removed another. It's recognized as 2 modifications and not 1 addition and deletion. The file diff in totality should make sense.